AENS.md

```diff
@@ -268,24 +268,30 @@ commitment := Hash(NameHash(name) + name_salt)
  ---------------- ----
 | name           | 63 |
  ---------------- ----
+| rent           | 32 |
+ ------------- -- ----
 | name_salt      | 32 |
  ---------------- ----
 ```
 
 Flow for a user:
 
-1. (optional) wait `n` blocks, s.t. that the block including the `pre-claim` cannot be reversed whp
+1. (optional) wait `n` blocks, s.t. that the block including the `pre-claim` cannot be reversed with high probability
 2. send `claim` transaction to reveal name and pay the associated fee
 
-If the time delta of `pre-claim` and `claim` is bigger than 300 blocks,
+If the time delta of `pre-claim` and `claim` is bigger than `floor(60 * 24 * 2/key_block_interval)` blocks,
 then the `claim` MUST be rejected.
 
-The `claim` transaction MUST be signed by the same private key as a
-`pre-claim` transaction containing a commitment to the name and nonce.
+The `claim` transaction MUST be signed by the same private key as a `pre-claim`
+transaction containing a commitment to the name and nonce.
 
 A `claim` transaction MUST NOT be in included in the same block as its
 `pre-claim`.
 
+`rent` is the reoccurring fee that has to be paid for names. Our first rent
```

Comment:
For how long is the name meant to be rented for? And is the user meant to be able to control that when claiming?

I see [code currently assumes relative TTL of 50000](https://github.com/aeternity/epoch/blob/e9e3b000475b58501fc0daf8e8b1d57ea8b25669/apps/aens/src/aens_claim_tx.erl#L140-L142) though not sure whether this is specified anywhere.